src/olympia/pages/templates/pages/shield_study_11.html

```diff
@@ -2,58 +2,176 @@
+                <li>
+                  {% trans %}
+                    Continue using Firefox as you normally would. Depending on how you use the Web, you may not notice any changes.
+                  {% endtrans %}
+                </li>
+                <li>
+                  {% trans %}
+                    After a few weeks, the study will end on its own. Your Firefox will go back to normal.
+                  {% endtrans %}
+                </li>
+                <li>
+                  {% trans %}
+                    We will ask you to fill out a quick optional survey to tell us about your experience. The survey should take less than 5 minutes to complete.
+                  {% endtrans %}
+                </li>
+            </ol>
+            <h2 id="leaving-the-study">
+              {% trans %}
+                Leaving the study
+              {% endtrans %}
+            </h2>
+            <p>
+              {% trans %}
+                The study will expire on its own in a few weeks. You may leave the study early. To do so, follow these steps:
+              {% endtrans %}
+            </p>
+            <ol>
+                <li>
+                  {% trans %}
+                    Type about:addons into the location bar, and press enter.
+                  {% endtrans %}
+                </li>
+                <li>
+                  {% trans %}
+                    Find Plugin Safety in the addons list.
+                  {% endtrans %}
+                </li>
+                <li>
+                  {% trans %}
+                    Click the remove button.
+                  {% endtrans %}
+                </li>
+                <li>
+                  {% trans %}
+                    If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.
+                  {% endtrans %}
+                </li>
+            </ol>
+            <p>
+              {% trans %}
+                Opting out of a study does not prevent you from participating in future studies.
+              {% endtrans %}
+            </p>
+            <h2 id="your-privacy">
+              {% trans %}
+                Your privacy
+              {% endtrans %}
+            </h2>
+            <h3 id="all-shield-studies">
+              {% trans %}
+                All Shield Studies
+              {% endtrans %}
+            </h3>
+            <p>
+              {% trans %}
+                Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.
+              {% endtrans %}
+            </p>
+            <p>
+              {% trans %}
+                This data is associated with <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a>.
+              {% endtrans %}
```

Comment:
Translated bodies of text should not include the URL. You need to pass it in as a variable so that it doesn't get garbled and so that it can be changed later. Example:
````
{% trans url='https://gecko.readthedocs.io/page/toolkit/components/telemetry/telemetry/index.html' %}
  This data is associated with <a href="{{ url }}">Firefox Telemetry collection</a>.
{% endtrans %}
````

Additionally, please note that the URL should be changed to the non-localized version which is `https://gecko.readthedocs.io/page/toolkit/components/telemetry/telemetry/index.html`